index.liquid

```diff
@@ -5,6 +5,14 @@ data:
   route: home
 ---
 
+<div id="modal">
```

Comment:
Back in my days it was too early to use `<dialog>` but nowadays it seems to have [great support](https://caniuse.com/?search=dialog), dialog elements are much better for modals, starting with the fact they come with the `.showModal()` function.

Reply:
Done, Im available for any other change needed,  hoping we can launch this landing page today! :)